### PR TITLE
e2e console: Mark conformance tests

### PR DESCRIPTION
**Pull request by fossedihelm**
### What this PR does

Mark conformance tests following https://github.com/kubevirt/kubevirt/blob/main/docs/conformance.md for console tests.
The two marked tests cover all the cases:
- the first one is the positive case with no concurrent serial console open, and possibility to connect more times.
- the second one is the negative case

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/compute/console.go [hunk 1]
```diff
@@ -79,21 +73,24 @@ var _ = SIGDescribe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@re
...
-				stream, err := virtClient.VirtualMachineInstance(vmi.Namespace).SerialConsole(vmi.Name, &kvcorev1.SerialConsoleOptions{})
+				stream, err := kubevirt.Client().VirtualMachineInstance(vmi.Namespace).SerialConsole(vmi.Name, &kvcorev1.SerialConsoleOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				defer stream.AsConn().Close()
 
 				firstConsoleErrChan := make(chan error, 1)
 				outReader, outWriter := io.Pipe()
 				inReader, _ := io.Pipe()
 				go func() {
+					defer GinkgoRecover()
```
Comment (0xFelix) [inline tests/compute/console.go:89]: This is only required if the goroutine can fail an assertion, no?

Comment (fossedihelm) [reply tests/compute/console.go:89]: Right, I think I messed up with something else

Comment (fossedihelm) [reply tests/compute/console.go]: Done, PTAL!

**Review by 0xFelix — COMMENTED** [review]
/approve
/retest

**Comment by brianmcarey** [comment]
/override pull-kubevirt-e2e-k8s-1.31-sig-performance

Triggered by status-reconciler when lane was bumped.

**Pull request merged**